helper/config/decode.go

```diff
@@ -145,3 +146,26 @@ func uint8ToStringHook(f reflect.Kind, t reflect.Kind, v interface{}) (interface
 
 	return v, nil
 }
+
+func updateString(oldString, newString string) string {
+	if oldString != "" {
+		if newString != "" {
+			return newString
+		}
+		return oldString
+	}
```

Comment:
🙂 This func could be simplified a bit to :
```go
if newString != "" {
    return newString
}
return oldString
```

Reply:
You're totally right.